internal/log/log.go

```diff
@@ -0,0 +1,118 @@
+// Package log implements a simple logger using only stdlib imports.
```

Comment:
i don't like having yet another logger nor do i see the reason to do this

Reply:
My reason was to not take a dependency because the clients are meant to run on shitty iot devices.. but yeah it's kinda pedantic. I'm too deep at this point tho

Reply:
i'm still with jared -- there's gotta be another tiny logger package out tehre to steal :/